dense_vector/challenges/default.json

```diff
@@ -36,7 +36,8 @@
       "name": "refresh-after-index",
       "operation": {
         "operation-type": "refresh",
-        "request-timeout": 1000
+        "request-timeout": 1000,
+        "include-in-reporting": true
```

Comment:
@pquentin It would be useful also do this `include-in-reporting` for `refresh-after-update`.